admin.php

```diff
@@ -20,6 +20,12 @@
 <h1>Admin Panel</h1>
 <div class="section-break"></div>
 
+<table class="new-project">
```

Comment:
why is this in a table? its just a button, not tabular data. please just use a div